core/error/error_macros.cpp

```diff
@@ -83,6 +88,9 @@ void _err_print_error(const char *p_function, const char *p_file, int p_line, co
 
 // Main error printing function.
 void _err_print_error(const char *p_function, const char *p_file, int p_line, const char *p_error, const char *p_message, bool p_editor_notify, ErrorHandlerType p_type) {
+	if (!error_log_enabled) {
```

Comment:
That looks like a hack. Not sure if it's necessary.
Though first scan errors aren't really useful, so it's not like this change is pointless.

Reply:
I used `CoreGlobals::print_error_enabled` instead. I found out about that yesterday.

Reply:
Nope, if I add a check on `CoreGlobals::print_error_enabled` in `_err_print_error`, a lot of unit test fail. I put back `error_log_enabled` but renamed it `print_error_enabled` to be more accurate.